### PR TITLE
Update default catalog 9.0.x

### PR DESCRIPTION
**Pull request by ps-jarvis**
| Questions         | Answers
| ----------------- | ------------------------------------------------------------
| Branch?           | 9.0.x
| Description?      | Automatic update of default catalog for 9.0.x
| Type?             | improvement
| Category?         | IN
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | ~
| UI tests.            | https://github.com/jolelievre/ga.tests.ui.pr/actions/runs/12115867567
| Fixed ticket?     | ~
| Related PRs       | ~
| Sponsor company   | ~

### CODE REVIEW

**Comment by Hlavtox** [comment]
Good thing that the translation extraction tool is fixed and there is only 5000 changed lines. 😄

**Review by tleon — DISMISSED** [review]

**Review by jolelievre — APPROVED** [review]

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
